tests/foreman/ui/test_bookmarks.py

```diff
@@ -259,10 +259,15 @@ def test_negative_create_with_duplicate_name(session, ui_entity, module_target_s
         existing_bookmark = session.bookmark.search(bookmark.name)[0]
         assert existing_bookmark['Name'] == bookmark.name
         ui_lib = getattr(session, ui_entity['name'].lower())
-        # this fails but does not raise UI error, BZ#1992652 closed wontfix
-        ui_lib.create_bookmark({'name': bookmark.name, 'query': query, 'public': True})
-        # assert there are no duplicate bookmarks
-        new_search = session.bookmark.search(bookmark.name)
-        assert len(new_search) == 1
-        # assert bookmark query wasn't overriden
-        assert new_search[0]['Search query'] == existing_bookmark['Search query']
+        # this fails but does not raise UI error in old style dialog, BZ#1992652 closed
+        # wontfix, but new style dialog raises error, both situations occur
+        try:
+            ui_lib.create_bookmark({'name': bookmark.name, 'query': query, 'public': True})
+        except (DisabledWidgetError, NoSuchElementException):
```

Comment:
So which dialogue will be used? Shouldn't we expect one of them? Shouldn't we expect an exception depending on what dialogue is being used? This looks too vague.

Reply:
> So which dialogue will be used? 

@lhellebr Hm, I didn't find a way to easily establish that, from the airgun point of view, they are the same. The differentiator is the behavior when trying to save bookmark with existing name

> Shouldn't we expect an exception depending on what dialogue is being used? This looks too vague.

In a way, I'm doing just that. If it doesn't raise, then I assume old style dialog and do additional checks to make sure duplicate stuff wasn't created.